test/remote/gateways/remote_cyber_source_test.rb

```diff
@@ -590,4 +604,11 @@ def test_verify_credentials
     assert !gateway.verify_credentials
   end
 
+  private
+
+  def assert_successful_response(response)
```

Comment:
It looks like this method wasn't extracted out from all of the other tests, leading to some tests still including all three lines (e.g., `test_successful_authorization_with_elo`) and others using this new private method. This seems like a good thing to make consistent.

Reply:
True, there were a lot of them. I refactored them all to DRY things out.